python-app/test/action.yml

```diff
@@ -6,15 +6,18 @@ inputs:
     description: "path to src directory"
     required: false
     default: "src"
+  report-format:
+    required: false
 
 runs:
   using: "composite"
   steps:
     - name: Run unittest
       run: |
-        pipenv run pytest -s --cov=./${SRC}
+        pipenv run pytest -s --cov=./${SRC} --cov-report=${FORMAT}
```

Comment:
there is no default so it expend to `--cov-report=`
?

Reply:
yep but it doesn't do anything if left empty